lib/tasks/performance_platform.rake

```diff
@@ -3,8 +3,8 @@ require 'performance_platform'
 namespace :performance_platform do
   desc 'Performance Platform transaction submission'
   task submit: :environment do
-    Rake::Task['performance_platform:submit_transactions'].invoke
     Rake::Task['performance_platform:submit_user_satisfaction'].invoke
+    Rake::Task['performance_platform:submit_transactions'].invoke
```

Comment:
So what was happening before? Was the call to `performance_platform:submit_transactions` stopping `performance_platform:submit_user_satisfaction` from being invoked?

Reply:
Ah no. I swapped these because I also fixed the fact that the tasks were running the wrong commands, and wanted to preserve the order they actually ran in. The fix for the mis-labelled tasks went in with the PP refactor.

Reply:
(see 50888c5)